frontends/web/src/routes/device/waiting.jsx

```diff
@@ -23,19 +23,21 @@ import { Shift, Alert } from '../../components/icon';
 import { Guide } from '../../components/guide/guide';
 import { Entry } from '../../components/guide/entry';
 import { Message } from '../../components/message/message';
+import Header from '../../components/header/Header';
 import Footer from '../../components/footer/footer';
 import { debug } from '../../utils/env';
 import InnerHTMLHelper from '../../utils/innerHTML';
 import * as style from './device.css';
 
-export default function Waiting({ testing }) {
+export default function Waiting(props) {
+    const { testing } = props;
     const title = i18n.t('welcome.title');
```

Comment:
convert this into a proper Component class with `@translate()`, to make `t` available in the props, to be consistent with the rest. 

I suspect that just using `i18n.t` directly does not work: switching language likely won't rerender this to change the text.